src/common/api.js

```diff
@@ -26,6 +27,8 @@ const axioInstance = axios.create({
   }
 });
 
+let technicClientBuild = '757'; // Requests to technic api need build id
```

Comment:
could you move this in constants?

Reply:
I'm afraid I can't. This is not a constant, only a placeholder valid at the time of writing this. The current build is subsequently fetched inside `getTechnicClientBuild`, which serves only as a global variable.

Reply:
> Global variable

Yeah you can put that into the constants

Reply:
I have added a constant with the '757' value but the variable is still needed for keeping the updated client build.